pr feedback (#987)
Graylog2/documentation: pages/alerts.rst

    Highlight the custom message with this `color <https://api.slack.com/reference/surfaces/formatting#quotes>`__ .

Custom Message

>     This is a message template and the data described above will be appended below the alert title, when the notification is sent.

Reviewer: ```suggestion
    The message that will be sent to Slack.  The data described above can be used in this template.
```

    This is a message template and the data described above will be appended below the alert title, when the notification is sent.

User Name (optional)
    Addressing a specific `user  <https://api.slack.com/reference/surfaces/formatting#mentioning-users>`__ in the slack workspace.

Icon URL

>     Image to use as the icon for this message.

Reviewer: Did you mean to delete Icon URL?

Author: good catch.
